ci-operator/config/konflux-workspaces/workspaces/konflux-workspaces-workspaces-main.yaml

```diff
@@ -0,0 +1,39 @@
+build_root:
+  project_image:
+    dockerfile_path: ci/Dockerfile
+releases:
+  initial:
+    integration:
+      name: "4.15"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.15"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 250Mi
+tests:
+- as: e2e
+  run_if_changed: ^.github/workflows/push.yaml$|^server|^operator|^ci
+  steps:
+    cluster_profile: aws-rhtap-qe
```

Comment:
I don't believe this is how the aws-rhtap-qe cluster_profile is being used by our team. @psturc @flacatus you might need to help me here but what I understand is that we exclusively use cluster_claims.

I think this because doing a [search for our cluster_profile](https://github.com/search?q=repo%3Aopenshift%2Frelease%20aws-rhtap-qe&type=code) shows that its not used in any config.

We are using [cluster_claim like this to get a cluster for the job](https://github.com/openshift/release/blob/93352c979a01cb17dcdcc30dd1dfb7463e56325e/ci-operator/config/konflux-ci/remote-secret/konflux-ci-remote-secret-main.yaml#L29) @sadlerap this is much different then how the interop team got clusters for testing that you've already seen.

Reply:
Fixed in a2e513179b8.